src/chrome/content/rules/Mint.com.xml

```diff
@@ -0,0 +1,12 @@
+<ruleset name="Mint.com" >
+	<target host="mint.com" />
+	<target host="www.mint.com" />
+	<target host="accounts.mint.com" />
+	<target host="help.mint.com" />
+	<target host="mobile.mint.com" />
+	<target host="wwws.mint.com" />
```

Comment:
```
credit.mint.com
```

Reply:
@J0WI for some reason it doesn't work for me :( (i asked a friend and he confirmed that both blog.mint.com and credit.mint.com worked for him so i added them)